app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Api
+  module V1
+    class RentsController < ApiV1Controller
+      def index
+        render_paginated Rent.rented_by(user_id).includes(:user, :book)
+      end
+
+      def create
+        rent = Rent.new(rent_params)
+        if rent.save
+          render json: rent, status: :created
+        else
+          render json: { errors: rent.errors.full_messages }, status: :unprocessable_entity
```

Comment:
You could have more generic error handler, with a rescue_from ;)

Reply:
Pretty useful! 
Thanks!!! 😃